docs/understanding-radicle/how-radicle-works.md

```diff
@@ -0,0 +1,334 @@
+---
+id: how-radicle-works
+title: How Radicle works
+---
+
+import Highlight from '@site/src/components/Highlight'
+
+At its most global, Radicle is a decentralized autonomous organization (DAO) solving challenges around sovereign code
+infrastructure and incentives for open-source work.
```

Comment:
I wouldn't characterize it this way, I think Radicle is a software stack, product and/or technology first and foremost. The Radicle DAO is the organization behind it.